msr-flows/Notes/type_1_flow.ipynb

```diff
@@ -0,0 +1,152 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Linear Growth Approximation\n",
+    "\n",
+    "#### Only suitable for SS\n",
+    "\n",
+    "The linear growth approximation is an approach which assumes that over some time step X, the growth is approximately linear. This means that the removal constant must be calculated such that the number of atoms at time X is the same as SaltProc at that time. As long as accuracy can be maintained to some position past where the SaltProc value is determined, then this method can be used to reduce the computational time of SaltProc by making a series of small linear approximations.\n",
+    "\n",
+    "![title](LGA.png)\n",
+    "\n",
+    "This approach has 3 main steps\n",
+    "\n",
+    "1. Run Serpent with no reprocessing for X days\n",
+    "\n",
+    "    1a. This gives enough data to create an approximate linear growth term\n",
+    "    \n",
+    "    $$\\frac{dN}{dt} = C$$\n",
+    "\n",
+    "    $$C = \\frac{N_f - N_i}{t_f - t_i}$$\n",
+    "\n",
+    "2. Plug the constant growth value in and then include reprocessing with the condition that it must match SaltProc at a given time. This is done by calculating waste atoms and solving with scipy.optimize root function.     \n",
+    "\n",
+    "    $$\\frac{dN}{dt} = C - \\lambda N$$\n",
+    "\n",
+    "    $$N(t) = N_i e^{-\\lambda t} + \\frac{C}{\\lambda} \\left( 1 - e^{-\\lambda t}  \\right)$$\n",
+    "    \n",
+    "    $$\\frac{dN_{waste}}{dt} = \\lambda N(t)$$\n",
+    "\n",
+    "    $$\\frac{dN_{waste}}{dt} = \\lambda (N_i e^{-\\lambda t} + \\frac{C}{\\lambda} \\left( 1 - e^{-\\lambda t}  \\right))$$\n",
+    "\n",
+    "    $$N_{waste}(t) = C_1 + Ct + \\frac{C e^{-\\lambda t}}{\\lambda} - N_0 e^{-\\lambda t}$$\n",
+    "\n",
+    "    $$C_1 = N_0 - \\frac{C}{\\lambda}$$\n",
+    "\n",
+    "3. Run Serpent using type 1 flows and the calculated reprocessing constants"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cycle Time Decay Approximation\n",
+    "\n",
+    "#### Suitable for BOC/SS/EOC\n",
+    "\n",
+    "This approximation treats the type 1 flow as a decay (which is how it is viewed mathematically). This means that the \"decay\" constant can be calculated from the \"half-life\" of a particular target.\n",
+    "\n",
+    "In this case, the half-lives are determined from the cycle times from Robertson et al. report on the MSBR that Andrei cites in the MSBR example.\n",
```

Comment:
>the half-lives are determined from the cycle times from Robertson et al. report on the MSBR that Andrei cites in the MSBR example.

This is a bit vague. Consider using the path to the file where the citation to the Robertson paper exists. Or if possible, include a link to the paper directly.